service/gcs/bridge/connection.go

```diff
@@ -5,48 +5,18 @@ import (
 
 	"github.com/pkg/errors"
 
+	"io"
```

Comment:
I definitely don't like the order that's in this file -- not sure how this happened.

I've been just letting goimports sort the imports when I save the file. By default, goimports will split things into Go runtime dependencies (io, etc.), then non-runtime dependencies (including internal ones), all sorted. This seems to look fine (IMO) and requires less manual intervention than the convention you propose:

```
	"encoding/binary"
	"io"

	"github.com/Microsoft/opengcs/service/gcs/prot"
	"github.com/Microsoft/opengcs/service/gcs/stdio"
	"github.com/Microsoft/opengcs/service/gcs/transport"
	"github.com/Microsoft/opengcs/service/libs/commonutils"
	"github.com/pkg/errors"
```

What do you think?